Source/Modules/delphi.cxx

```diff
@@ -5769,10 +5794,10 @@ class DELPHI:public Language {
 
 		  variable_name =GetattrNew(n, "sym:name");
 
-		  wrapping_member_flag = true;
+		  //wrapping_member_flag = true;
 		  static_flag = true;
 		  Language::staticmembervariableHandler(n);
-		  wrapping_member_flag = false;
+		  //wrapping_member_flag = false;
```

Comment:
What is the purpose of commenting out setting of the `wrapping_member_flag`? If this is really needed, why do not delete the code?